### PR TITLE
Recipe for jansson

### PR DESCRIPTION
**Pull request by mariusvniekerk**
This is a c dependency for avro 1.8+

### CODE REVIEW

File: recipes/jansson/build.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+./configure --prefix=$PREFIX
+make
+make install
```
Comment (ocefpaf) [inline recipes/jansson/build.sh]: Is there a make check that we can run here?

Comment (mariusvniekerk) [inline recipes/jansson/build.sh]: I did a simple test against libjansson.a

Comment (mariusvniekerk) [inline recipes/jansson/build.sh]: Guess i can just add a make check

**Comment by jakirkham** [comment]
@akheron, just letting you know we have added `jansson` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) (`conda install -c conda-forge jansson`). The maintenance burden here is light as all testing releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this repo]( https://github.com/conda-forge/jansson-feedstock ). If you have any interest in being added as a maintainer (to make new releases for instance), we would love to have you on board. Thanks for your work on `jansson`. 😄

**Pull request merged**